.github/workflows/development.yml

```diff
@@ -0,0 +1,43 @@
+name: Development Build and Heroku Push
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: akhileshns/heroku-deploy@v3.12.12 
+      with:
+        heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+        heroku_app_name: ${{secrets.HEROKU_APP_NAME}}
+        heroku_email: "marklawrence@hotmail.co.uk"
+        justlogin: true
```

Comment:
Not sure why this wasn't working for you for the actual deployment, I think it should _just work_ when the `useDocker` flag is on, but no issue with just doing the steps below!